### PR TITLE
completed homepage design and functionality

### PR DESCRIPTION
**Pull request by prakritip0**
# Summary of change

* Added Navbar
* Created the Homepage design and functionality

## Type of change

- [x] New feature

# How Has This Been Tested?

- [x] Manual testing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: .DS_Store
Comment (sarad-singh) [inline .DS_Store:1]: See .gitignore, and use that to ignore such files

File: index.html [hunk 1]
```diff
@@ -5,13 +5,206 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Task 1</title>
     <link rel="stylesheet" href="styles.css">
+    <title>Task 1</title>
```
Comment (sarad-singh) [inline index.html]: Title specific to the page would be more friendly

File: index.html [hunk 2]
```diff
@@ -5,13 +5,206 @@
...
 </head>
```
Comment (sarad-singh) [inline index.html:12]: To add title icon:
`<link rel="icon" type="image/x-icon" href="/images/favicon.ico">`

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,394 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700&family=Outfit:wght@100;400&display=swap');
```
Comment (sarad-singh) [inline styles.css]: Dividing styles into different files for different pages would be better

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    max-height: 100vh;
```
Comment (sarad-singh) [inline styles.css]: See This: https://web.dev/learn/css/the-cascade/

**Comment by sarad-singh** [comment]
Add some description on PR description section, summarizing overall changes that this PR would bring to your project/main-branch after this branch gets merged.

Try to add more commits in further PR's 
for example:
1. feat: Add SideNavbar
2. feat: Add hover effects for navbar links
3. feat: Add page title section
4. feat: Add homepage Cards
6. feat: "Add" button on homepage
and so on

**Review by bisheshbh — APPROVED** [review]

**Review by sarad-singh — APPROVED** [review]

**Pull request merged**